CMakeLists.txt

```diff
@@ -19,6 +19,27 @@ project(velox)
 
 option(VELOX_BUILD_TESTING "Enable Velox tests" ON)
 option(VELOX_ENABLE_DUCKDB "Build duckDB to enable differential testing." ON)
+option(VELOX_ENABLE_EXEC "Build exec." ON)
+option(VELOX_ENABLE_AGGREGATES "Build aggregates." ON)
+option(VELOX_ENABLE_HIVE_CONNECTOR "Build Hive connector." ON)
+option(VELOX_ENABLE_PRESTO_FUNCTIONS "Build Presto SQL functions." ON)
+option(VELOX_ENABLE_SPARK_FUNCTIONS "Build Spark SQL functions." ON)
+option(VELOX_ENABLE_EXPRESSION "Build expression." ON)
+option(VELOX_ENABLE_PARSE "Build parse." ON)
+option(VELOX_ENABLE_EXAMPLES "Build examples." ON)
+
+if(${VELOX_DEFAULT})
```

Comment:
not sure if we settled on the name yet, but seems to me that by default we should just build the whole thing. So maybe this flag could be named "VELOX_BASE" or "VELOX_MINIMAL" ?

Reply:
> not sure if we settled on the name yet, but seems to me that by default we should just build the whole thing. So maybe this flag could be named "VELOX_BASE" or "VELOX_MINIMAL" ?

VELOX_DEFAULT is the results from our discussion earlier so I started with the name. The original idea is to provide a default values to a build option if it's not set. But somehow we use "option" to define a variable and provide the default value already so the purpose of VELOX_DEFAULT become what it is now to just trigger a base or minimal build. I like VELOX_BASE and I can change the files accordiningly.